odoolint/tests/test_convert_file.py

```diff
@@ -27,7 +29,7 @@ def filter(self, record):
         self.buffer.append(record.__dict__)
 
 
-class TestConvertFile(common.TransactionCase):
+class TestConvertFile(common.TransactionCase, unittest.TestCase):
```

Comment:
I mean
I think `common.TransactionCase` is inherited from `common.TransactionCase`